docs/index.rst

```diff
@@ -52,6 +54,43 @@ Why MembraneCurvature?
 in Python 3 to derive 2D maps of membrane curvature from MD Simulations, using the most recent version of `MDAnalysis`_ 
 Are you interested in calculating mean and Gaussian curvature from MD simulations? This tool is for you!
 
+
+Installation
+--------------
+
+The main dependency in MembraneCurvature is `MDAnalysis`_. You can find
+instructions to install the latest stable version of MDAnalysis via `conda`_ in the `User Guide`_.
```

Comment:
Could you link directly to the installation notes in the user guide?

Reply:
Sure, I'll fix it! :)